cernopendata/modules/fixtures/data/records/cms-primary-datasets.json

```diff
@@ -637,7 +637,7 @@
       "attribution": "CC0"
     },
     "methodology": {
-      "description": "Dedicated primary dataset for detector monitoring. Events stored in this primary dataset were selected because of presence of at least one high-energy electron in the event, but the acceptance criteria for these electrons are different than those of the Electron primary dataset."
```

Comment:
Ditto here: `<p>Dedicated primary... of the Electron primary dataset.</p><p><strong>HLT trigger paths</strong>...</p>`

Reply:
Thank you, fixed that as well and all other cases